src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/ContributorTeamRepository.cs

```diff
@@ -65,17 +66,19 @@ public List<ContributorTeam> Find()
         /// <param name="id">The id of contributor team to remove.</param>
         public void Remove(int id)
         {
-            throw new NotImplementedException();
+            var contributorTeamDelete = _unitOfWork.Context.ContributorTeams.Single(x => x.Id == id);
+            _unitOfWork.Context.ContributorTeams.Remove(contributorTeamDelete);
+            //TODO: make unit test
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1135)